### PR TITLE
eployee and service lists added to saloon profile page. employee add …

### PR DESCRIPTION
**Pull request by assalaa**
…and service add functions also created

### CODE REVIEW

File: lib/database/database_image.dart [hunk 1]
```diff
@@ -13,18 +13,21 @@ import 'package:permission_handler/permission_handler.dart';
 class DatabaseImage {
   Future<String> uploadImage(XFile image, String path) async {
     if (kDebugMode) {
-      path = 'test/$path';
+      path = 'images/$path';
```
Comment (arielsam567) [inline lib/database/database_image.dart]: change to test

**Review by recepsenoglu — DISMISSED** [review]
ok

File: lib/database/database_image.dart (hunk 1 above)
Comment (recepsenoglu) [reply lib/database/database_image.dart]: ok

**Review by recepsenoglu — APPROVED** [review]
ok

**Review by recepsenoglu — APPROVED** [review]
ok

**Review by arielsam567 — APPROVED** [review]
ok

**Pull request merged**